### PR TITLE
Updated data/channels.csv

### PR DESCRIPTION
**Pull request by DerpyMcDonuts**
Added WTLWDT2.us, WRDQDT1.us, WEFSDT1.us, and WEFSDT2.us
Updated KWGNDT1.us and KWGNDT2.us

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -28046,6 +28047,7 @@ WRDCDT2.us,WRDC-DT2,,CHARGE!,US,US-NC,Durham,s/US-NC,eng,,FALSE,,,,,https://uplo
 WRDCDT3.us,WRDC-DT3,,Comet TV,US,US-NC,Durham,s/US-NC,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/9/97/Comet_logo.svg/512px-Comet_logo.svg.png
 WRDELD1.us,WRDE-LD1,,NBC,US,US-MD,Salisbury,s/US-MD,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/c/c7/NBC_2013_fixed_logo.png/512px-NBC_2013_fixed_logo.png
 WRDELD2.us,WRDE-LD2,,MyNetworkTV,US,US-MD,Salisbury,s/US-MD,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/MyNetworkTV_2D_Logo.svg/512px-MyNetworkTV_2D_Logo.svg.png
+WRDQDT1.us,WRDQ,,,US,US-FL,Orlando,s/US-FL,eng,,FALSE,,,,https://www.wftv.com/tv27community/,https://upload.wikimedia.org/wikipedia/en/d/da/Wrdq_2009.png
```
Comment (freearhey) [inline data/channels.csv:28050]:
```suggestion
WRDQDT1.us,WRDQ-DT1,,,US,US-FL,Orlando,s/US-FL,eng,,FALSE,,,,https://www.wftv.com/tv27community/,https://upload.wikimedia.org/wikipedia/en/d/da/Wrdq_2009.png
```

File: data/channels.csv [hunk 2]
```diff
@@ -10839,8 +10839,7 @@ KHSVDT2.us,KHSV-DT2,,Decades,US,US-NV,Las Vegas,s/US-NV,eng,,FALSE,,,,,https://u
 KHSVDT4.us,KHSV-DT4,,Antenna,US,US-NV,Las Vegas,s/US-NV,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/2/2c/Antenna_TV_logo.svg/512px-Antenna_TV_logo.svg.png
 KHSVDT5.us,KHSV-DT5,,Start TV,US,US-NV,Las Vegas,s/US-NV,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/0/06/Start_TV_logo.svg/512px-Start_TV_logo.svg.png
 KHTMLP1.us,KHTM-LP1,,Hillsong,US,US-TX,Lufkin,s/US-TX,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/b/b0/Hillsong_Channel_Logo.png/512px-Hillsong_Channel_Logo.png
-KHTVCD1.us,KHTV-CD1,,Infomercials,US,US-CA,Los Angeles,s/US-CA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/As_seen_on_TV.svg/512px-As_seen_on_TV.svg.png
-KHTVCD4.us,KHTV-CD4,,Jewelry Television,US,US-CA,Los Angeles,s/US-CA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/5/50/Jewelry_television_logo.png
+KHTVCD1.us,KHTV-CD,,MeTV+,US,US-CA,Los Angeles,s/US-CA,eng,,FALSE,,,,https://www.metv.com/metvplus/,https://i.imgur.com/ubCl0S9.png
```
Comment (freearhey) [inline data/channels.csv:10842]:
```suggestion
KHTVCD1.us,KHTV-CD1,,MeTV+,US,US-CA,Los Angeles,s/US-CA,eng,,FALSE,,,,https://www.metv.com/metvplus/,https://i.imgur.com/ubCl0S9.png
```

File: data/channels.csv [hunk 3]
```diff
@@ -12938,8 +12937,8 @@ KTWODT2.us,KTWO-DT2,,Cozi TV,US,US-WY,Casper,s/US-WY,eng,,FALSE,,,,,https://uplo
 KTWUDT1.us,KTWU-DT1,,PBS,US,US-KS,Topeka,s/US-KS,eng,,FALSE,,,,http://ktwu.wuacc.edu/,https://upload.wikimedia.org/wikipedia/en/thumb/3/33/PBS_logo.svg/512px-PBS_logo.svg.png
 KTWUDT2.us,KTWU-DT2,,PBS World,US,US-KS,Topeka,s/US-KS,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/6/6b/PBSworld.png
 KTWUDT3.us,KTWU-DT3,,PBS Enhance,US,US-KS,Topeka,s/US-KS,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/3/33/PBS_logo.svg/512px-PBS_logo.svg.png
-KTXADT1.us,KTXA-DT1,,KTXA,US,US-TX,Dallas,s/US-TX,eng,,FALSE,,,,https://dfw.cbslocal.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/7d/KTXA_2012.png/512px-KTXA_2012.png
-KTXADT2.us,KTXA-DT2,,CBSN,US,US-TX,Fort Worth,s/US-TX,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/d/dc/CBS_News_logo_%282020%29.svg/512px-CBS_News_logo_%282020%29.svg.png
+KTXADT1.us,KTXA,,,US,US-TX,Fort Worth,s/US-TX,eng,,FALSE,,,,https://dfw.cbslocal.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/7d/KTXA_2012.png/512px-KTXA_2012.png
```
Comment (freearhey) [inline data/channels.csv:12940]:
```suggestion
KTXADT1.us,KTXA-DT1,,,US,US-TX,Fort Worth,s/US-TX,eng,,FALSE,,,,https://dfw.cbslocal.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/7d/KTXA_2012.png/512px-KTXA_2012.png
```

File: data/channels.csv [hunk 4]
```diff
@@ -25688,7 +25687,9 @@ WEEKDT1.us,WEEK-DT1,,NBC,US,US-IL,Bloomington,s/US-IL,eng,,FALSE,,,,http://www.w
 WEEKDT2.us,WEEK-DT2,,ABC,US,US-IL,Peoria,s/US-IL,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/ABC-2021-LOGO.svg/512px-ABC-2021-LOGO.svg.png
 WeekDT3.us,Week-DT3,,CW,US,US-IL,Peoria,s/US-IL,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/The_CW.svg/512px-The_CW.svg.png
 WeeTV.kr,WeeTV,,,KR,,,c/KR,kor,,FALSE,,,,http://tra.co.kr/web_kor2/,https://i.imgur.com/AQbjUyc.png
-WEFSDT3.us,WEFS-DT3,,NASA,US,US-FL,Orlando,s/US-FL,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/b/b4/NASA_TV.svg/512px-NASA_TV.svg.png
+WEFSDT1.us,WEFS,,,US,US-FL,Cocoa,s/US-FL,eng,,FALSE,,,,https://www.easternflorida.edu/community-resources/wefs-tv/,https://i.imgur.com/547tKef.png
```
Comment (freearhey) [inline data/channels.csv:25690]:
```suggestion
WEFSDT1.us,WEFS-DT1,,,US,US-FL,Cocoa,s/US-FL,eng,,FALSE,,,,https://www.easternflorida.edu/community-resources/wefs-tv/,https://i.imgur.com/547tKef.png
```

**Comment by DerpyMcDonuts** [comment]
Corrected from freearhey's suggestions. Got the convention down now.

**Review by freearhey — APPROVED** [review]

**Pull request merged**